contentrepo/urls.py

```diff
@@ -71,7 +70,7 @@
     #    path("pages/", include(wagtail_urls)),
 ]
 
-urlpatterns += i18n_patterns(
-    path("search/", search_views.search, name="search"),
-    path("", include(wagtail_urls)),
-)
+# urlpatterns += i18n_patterns(
+#     path("search/", search_views.search, name="search"),
+#     path("", include(wagtail_urls)),
+# )
```

Comment:
Code that's commented out should either be deleted, or have an explanation of why it hasn't been deleted. In this case, I think let's just delete it.